### PR TITLE
Tech talks layout, dictionary items, table CSS update.

### PR DESCRIPTION
**Pull request by Pete-Robelou**
- Community tech talks layout
- Dictionary items
- Table CSS update

### CODE REVIEW

File: src/en-GB/community/tech-talks/index.html [hunk 1]
```diff
@@ -3,52 +3,89 @@
...
+        <tbody>
+          {% for item in item.results %}
+          <tr>
+            <td class="w-36">
+              <time datetime="{{ item.date }}">{{ item.date | formatDate(locale) }}</time>
+            </td>
+            <td>
+              {% if item.url %}
+              <a class="a" href="{{ item.url }}" target="_blank" rel="noreferrer noopener">{% endif %}{{ item.title }}{% if item.url %}</a
+              >{% endif %} {% if item.presenter %}<span class="inline-block text-lower">{{ 'with' | i18n }} {{ item.presenter }}</span>{%
```
Comment (adamduncan) [inline src/en-GB/community/tech-talks/index.html:79]: @Pete-Robelou Is there a reason we `text-lower`'d this? The `tech_talks` content (data) is properly cased with names etc. Perhaps we render as is?

Comment (Pete-Robelou) [reply src/en-GB/community/tech-talks/index.html:79]: It was to lowercase the word `with` as grammatically the whole line reads as a sentence. I guess we could just wrap a `span` around the word `with` so only that is lowercase ?

Screenshot 2021-06-23 at 11 07 54
Screenshot 2021-06-23 at 11 08 07

Comment (adamduncan) [reply src/en-GB/community/tech-talks/index.html:79]: Ah, gotcha. Makes sense. Yeah, good shout. Let's just apply that to `with` 👍

Comment (Pete-Robelou) [reply src/en-GB/community/tech-talks/index.html:79]: I've removed the `<span class="inline-block">` as this was originally to improve the wrapping on smaller screens, however in retrospect it's not really an improvement 🙂 . 

We're now only wrapping `with` to make it lowercase.

**Pull request merged**